ui/js/controllers/filters.js

```diff
@@ -91,15 +91,13 @@ treeherderApp.controller('JobFilterCtrl', [
 
         $scope.pinAllShownJobs = function () {
             if (!thPinboard.spaceRemaining()) {
-                thNotify.send("Pinboard is full.  Can not pin any more jobs.",
-                    "danger",
-                    true);
+                thNotify.send(thPinboardCountError, 'danger', true);
```

Comment:
We recently changed the format of the ``.send`` function so that it doesn't take a boolean as the last argument, but an ``options`` object.  So this should be changed to:

    thNotify.send(thPinboardCountError, 'danger', { sticky: true });

Sorry I didn't catch this the first time around.  And I realize this was old code that we'd somehow missed in the syntax change.  But let's fix it here anyway.  Thanks!  :)

Reply:
Fixed!